.github/workflows/ci.yaml

```diff
@@ -1232,18 +1222,7 @@ jobs:
         uses: actions/download-artifact@v4.1.4
```

Comment:
above here is still a `Upload test coverage to Codecov (partial suite)` do we want to keep the name like that?

Reply:
I would say yes as we have two codecov jobs